linkcheck/models.py

```diff
@@ -329,7 +329,12 @@ def _check_external(self, tested_url, external_recheck_interval):
                 # This means it could not follow the redirection
                 self.status = False
```

Comment:
Just out of curiosity: Can this case even be reached?
If something goes wrong with the redirected url (timeout, connection error, too many redirects etc.), requests throws an exception anyway. So I don't think there is a case where the final status code is still a redirect, requests did not follow it but did not raise an exception...

Reply:
Ah ok, got it: When the response server is broken in a way that it returns status code 30x, but does not provide a `Location` header, this case is indeed reached. Hopefully there aren't many servers which do that, but as a checker, we should probably consider this case.